portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -15,33 +15,66 @@
 package com.google.sps.servlets;
 
 import com.google.gson.Gson;
+import com.google.sps.data.Comment;
+import sun.tools.jconsole.JConsole;
 
 import java.io.IOException;
 import javax.servlet.annotation.WebServlet;
 import javax.servlet.http.HttpServlet;
 import javax.servlet.http.HttpServletRequest;
 import javax.servlet.http.HttpServletResponse;
 
-import java.util.Arrays;
+import java.time.LocalDateTime;
+import java.util.ArrayList;
 import java.util.List;
 
 /** Servlet that returns some example content.*/
 @WebServlet("/data")
```

Comment:
Since this webservlet now only serve comment data, should we change the /data to /comment?